src/core/client.ts

```diff
@@ -72,10 +84,15 @@ export class OakConsentClient {
     if (typeof window !== "undefined") {
       window.oakConsent = this;
     }
-    this.isReady = this.init();
   }
 
   public async init() {
+    // Only initialize once
+    if (this.isInitialized) {
+      return;
+    }
```

Comment:
This was only necessary because it is possible for the provider to be unmounted between renders and calling this twice is undesirable.